src/D2L.CodeStyle.Analyzers/Immutability/ImmutabilityContext.Factory.cs

```diff
@@ -77,24 +78,24 @@ internal partial class ImmutabilityContext {
 			("System.IO.Abstractions.FileSystem", default)
 		);
 
+
+		internal static readonly ImmutableArray<(string TypeName, string MethodName, string AssemblyName)> KnownImmutableReturningMethods = ImmutableArray.Create(
+			( "System.Array", "Empty", default(string) ),
+			( "System.Linq.Enumerable", "Empty", default(string) )

```

Comment:
```suggestion
			( "System.Array", "Empty", default ),
			( "System.Linq.Enumerable", "Empty", default )
```

Reply:
Can't do.
At least one of them has to have the `(string)` so that the rest can infer the type. The other `ImmutableArray` works because some of them are defined, which tells the rest that they are strings.

Reply:
Could probably do via the `Create<(string, string, string)>`, but, not important.